package.json

```diff
@@ -302,6 +304,7 @@
     "nonce-tracker": "^1.0.0",
     "obj-multiplex": "^1.0.0",
     "pify": "^5.0.0",
+    "playwright": "^1.29.2",
```

Comment:
Could you move both of these from `dependencies` to `devDependencies`? We only use `dependencies` for things that end up in the built extension

Reply:
Thanks for the review Mark, I will create a follow up PR to add these changes :)